src/app/shared/services/user/identyum.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@angular/core';
+import { BackendHttpClient } from '../backend-http-client.service';
+import { User } from './signup.service';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class IdentyumService {
+    constructor(private http: BackendHttpClient) {
+    }
+
+    getSessionID() {
+        return this.http.get<IdentyumClientToken>('/api/user/identyum/token');
+    }
+
+    verifyUser(sessionState: string) {
+        return this.http.post<User>('/api/user/me/verify', <VerifyUserData>{
```

Comment:
Probably it would be good to rename this route to `/api/user/identyum/verify`?

Reply:
Sure, may I change it?